docker/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM ubuntu:18.04
```

Comment:
Thanks @commandlinegirl! I tend to use the python slim-stretch base images (https://hub.docker.com/_/python) as they are quite a bit smaller than Ubuntu. Any reason Ubuntu has to be used here?